queries.sql

```diff
@@ -15,7 +15,7 @@ UPDATE animals SET species = '';
 
 START Transaction;
 UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
-UPDATE animals SET species = 'pokemon' WHERE species LIKE '';
+UPDATE animals SET species = 'pokemon' WHERE species = "";
```

Comment:
- [ ] Before and after every query you should Verify that change were made.

Reply:
DONE 👍 

![Screenshot 2022-10-20 at 3 06 56 AM](https://user-images.githubusercontent.com/30742617/196809500-a4d6213e-6177-4540-80fd-ee28c13aafe2.png)